pwiz_tools/Skyline/Model/Lib/Library.cs

```diff
@@ -34,13 +35,15 @@
 using pwiz.Skyline.Model.Crosslinking;
 using pwiz.Skyline.Model.DocSettings;
 using pwiz.Skyline.Model.DocSettings.Extensions;
+using pwiz.Skyline.Model.Hibernate;
 using pwiz.Skyline.Model.Irt;
 using pwiz.Skyline.Model.Lib.ChromLib;
 using pwiz.Skyline.Model.Lib.Midas;
 using pwiz.Skyline.Model.Prosit;
 using pwiz.Skyline.Model.Results;
 using pwiz.Skyline.Model.RetentionTimes;
 using pwiz.Skyline.Properties;
+using pwiz.Skyline.SettingsUI;

```

Comment:
This "using" statement is causing the CodeInspectionTest failure.
That test expects there to be exactly 3 places in the model code where it is using something from a UI namespace.
This particular line ends up causing the number of violations to be 4, which causes the test failure.

Reply:
Any idea why this file is pulling in that "using"? I guess maybe the ViewLibraryPepInfo.

Reply:
One way to make the messages clearer is to actually clean up all of these in the existing code and then report only new ones. I think you own a fair number of the remaining cases, Nick.